dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.config-3.1.feature

```diff
@@ -1,8 +1,8 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.org.eclipse.microprofile.config-3.1
 singleton=true
--features=io.openliberty.mpCompatible-6.1, \
-  io.openliberty.jakarta.cdi-4.0
+# io.openliberty.mpCompatible-x.x comes from io.openliberty.org.eclipse.microprofile.config.3.1.ee features
+-features=io.openliberty.org.eclipse.microprofile.config.3.1.ee-10.0; ibm.tolerates:="11.0"
```

Comment:
Why is it necessary to split the private mpConfig-3.1 API feature into two internal private features?